rotkehlchen/chain/arbitrum_one/modules/umami/balances.py

```diff
@@ -0,0 +1,220 @@
+                        balance=balance,
+                        balances=balances,
+                        vault_token=vault_token,
+                        user_address=user_address,
+                    )
+            except RemoteError as e:
+                log.error(f'Failed to query umami balances for vault {vault_token.evm_address}: {e!s}')  # noqa: E501
+                continue
+
+        # get staked balances.
+        for user_address in addresses_with_deposits:
+            try:
+                results = self.evm_inquirer.multicall(
+                    calls=[
+                        (
+                           self.umami_masterchef_contract.address,
+                           self.umami_masterchef_contract.encode(
+                               method_name='userInfo',
+                               arguments=[idx, user_address],
+                           ),
+                        ) for idx in range(4)
+                    ],
+                    call_order=call_order,
+                    calls_chunk_size=3,  # etherscan throws an error for >3 chunk size.
+                )
+                if not results:
+                    log.error(f'Failed to query umami vault balances for address {user_address}')
+                    continue
+
+                for idx, result in enumerate(results):
+                    balance, _ = self.umami_masterchef_contract.decode(
+                        result=result,
+                        method_name='userInfo',
+                        arguments=[idx, user_address],
+                    )
+
+                    if balance == 0:
+                        continue
+
+                    gm_vault_token_address = gm_vault_addresses[idx]
+                    gm_vault_token = GlobalDBHandler().get_evm_token(
+                        address=gm_vault_token_address,
+                        chain_id=self.evm_inquirer.chain_id,
+                    )
+                    if gm_vault_token is None:
+                        log.error(f'Missing vault {gm_vault_token_address} from DB, cannot process vault balance. Skipping')  # noqa: E501
+                        continue
+
+                    self._process_vault_balance(
+                        balance=balance,
+                        balances=balances,
+                        user_address=user_address,
+                        vault_token=gm_vault_token,
+                    )
+            except RemoteError as e:
+                log.error(f'Failed to query umami balances for address {user_address}: {e!s}')
+                continue
+
+        return balances
+
+    def _process_vault_balance(
+            self,
+            balance: int,
+            vault_token: 'EvmToken',
+            balances: 'BalancesSheetType',
+            user_address: 'ChecksumEvmAddress',
+    ) -> None:
+        """Process vault balance and add to user's balance sheet.
+
+        Args:
+            vault_token: The vault token
```

Comment:
keep arguments in docstring same order as arguments in function please. But do you even need succh a docstring? This is just reading out the name of the variable. The point of good argument naming and typing is to not need these kind of docstrings